python/module/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+install(FILES "openstudio.py" DESTINATION Python COMPONENT "Python")
+
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/openstudio.py ${PROJECT_BINARY_DIR}/Products/python/openstudio.py COPYONLY)
+
+foreach(p ${ALL_PYTHON_BINDING_TARGETS})
+  target_link_libraries(${p}
+    PRIVATE
+    openstudiolib
+    Python::Python
+  )
+endforeach()
+
+# convenience target to build the SDK
+add_custom_target(python_sdk ALL
+  DEPENDS
+    ${ALL_PYTHON_BINDING_TARGETS}
+    ${ALL_PYTHON_GENERATED_SRCS}
+)
+
+
+if (BUILD_PYTHON_PIP_PACKAGE)
+
+  # configure_file(${CMAKE_CURRENT_SOURCE_DIR}/openstudio_dynamic.py ${PROJECT_BINARY_DIR}/Products/python/openstudio_dynamic.py COPYONLY)
+  configure_file(${CMAKE_CURRENT_SOURCE_DIR}/openstudio.py ${PYTHON_PACKAGE_FOLDER}/openstudio/__init__.py COPYONLY)
+
+  if(PYTHON_PIP_REPOSITORY STREQUAL "testpypi")
+    execute_process( COMMAND ${Python_EXECUTABLE} "${CMAKE_CURRENT_SOURCE_DIR}/find_pypi_tag.py" OUTPUT_VARIABLE PYPI_VERSION)
```

Comment:
Path is wrong, find_pypi_tag.py is one level above currently.

Reply:
Haven't tested the pip package. The could be plenty of issues introduced by my rearrangement of the deck chairs.

Reply:
Fixed this specific issue in c6fa66eed9bbe1f0b44cde8cce96fb4dbbfc76ce

Reply:
Awesome. Thanks!